OpenRA.Mods.Common/Activities/Resupply.cs

```diff
@@ -187,7 +187,7 @@ void OnResupplyEnding(Actor self, bool isHostInvalid = false)
 			{
 				if (wasRepaired || isHostInvalid || (!stayOnResupplier && aircraft.Info.TakeOffOnResupply))
 				{
-					if (self.CurrentActivity.NextActivity == null && rp != null && rp.Path.Count > 0)
+					if (self.CurrentActivity.NextActivity == null && rp != null && rp.Path.Any())
```

Comment:
Why this change? Avoiding LINQ should be good for performance?

Reply:
You're right, I was reflex confusing this with `Count()`.